### PR TITLE
fix issue

### PR DESCRIPTION
**Pull request by ttt-77**

### CODE REVIEW

**Comment by ttt-77** [comment]
- [ ] Remove the tests for UDF aggregation exact queries due to inadequate support for this feature. Instead, raise an exception for it.
- [ ] Fix query blob table extraction.
- [ ]Modify the 'execute_user_defined_function' to returns a DataFrame.

**Comment by ttt-77** [comment]
@ddkang Could you please review this PR?

File: aidb/engine/base_engine.py [hunk 1]
```diff
@@ -445,8 +445,9 @@ def execute_user_defined_function(
       self,
       res_df: pd.DataFrame,
       dataframe_sql: Dict,
-      query: Query
-  ) -> List[Tuple]:
+      query: Query,
+      additional_select_col: List[str] = []
```
Comment (ddkang) [inline aidb/engine/base_engine.py:449]: What is this actually for? The comment needs more context

File: aidb/query/query.py [hunk 2]
```diff
@@ -748,6 +752,9 @@ def check_udf_query_validity(self):
             and expression.args['this'] in self.config.user_defined_functions):
           raise Exception("We don't support nested user defined function currently")
 
+    if self._expression.find(exp.AggFunc) is not None:
+      raise Exception("We don't support user defined function for aggregation query")
```
Comment (ddkang) [inline aidb/query/query.py:756]: AIDB does not...

**Review by ddkang — APPROVED** [review]

**Pull request merged**